frontend/sw360-portlet/src/main/webapp/html/projects/ajax/linkedProjectsAjax.jsp

```diff
@@ -23,36 +22,28 @@
 <%@ page import="org.eclipse.sw360.datahandler.thrift.projects.ProjectRelationship" %>
 <jsp:useBean id="projectList" type="java.util.List<org.eclipse.sw360.datahandler.thrift.projects.ProjectLink>"  scope="request"/>
 
-
 <core_rt:forEach items="${projectList}" var="projectLink" varStatus="loop">
     <core_rt:set var="uuid" value="${projectLink.id}"/>
     <tr id="projectLinkRow${uuid}" >
         <td width="32%">
             <input type="hidden" value="${projectLink.id}" name="<portlet:namespace/><%=Project._Fields.LINKED_PROJECTS%><%=ProjectLink._Fields.ID%>">
-            <label class="textlabel stackedLabel" for="projectName">Project name</label>
             <input id="projectName" type="text" class="toplabelledInput" placeholder="Enter project"
                    value="<sw360:out value="${projectLink.name}"/>" readonly onclick="window.location='<sw360:DisplayProjectLink projectId="${projectLink.id}" bare="true"/>'"/>
         </td>
         <td width="32%">
             <input type="hidden" value="${projectLink.version}" name="<portlet:namespace/><%=ProjectLink._Fields.VERSION%>">
-            <label class="textlabel stackedLabel" for="projectVersion">Project version</label>
             <input id="projectVersion" type="text" class="toplabelledInput" placeholder="No project version"
```

Comment:
This and similar fields all have the class `toplabelledInput`, but now you removed their labels. Does it have any implications other than a little inconsistency in code?

Reply:
indeed they could lead to confusion. I removed the ´toplabelledInput´ class attributes